src/modules/user/user.usecase.ts

```diff
@@ -698,6 +700,72 @@ export class UserUseCases {
     });
   }
 
+  async sendAccountUnblockEmail(email: User['email']): Promise<void> {
+    const secret = this.configService.get('secrets.jwt');
+    const user = await this.userRepository.findByEmail(email);
+
+    if (!user) {
+      throw new NotFoundException();
+    }
+
+    const defaultIat = getTokenDefaultIat();
+
+    await this.userRepository.updateByUuid(user.uuid, {
+      lastPasswordChangedAt: new Date(defaultIat * 1000),
+    });
+
+    const unblockAccountToken = SignWithCustomDuration(
+      {
+        payload: {
+          uuid: user.uuid,
+          email: user.email,
+          action: AccountTokenAction.Unblock,
+        },
+        iat: defaultIat,
+      },
+      secret,
+      '48h',
+    );
+
+    const driveWebUrl = this.configService.get('clients.drive.web');
+    const unblockAccountTemplateId = this.configService.get(
```

Comment:
We need to use mail limits here (10 per day is fine), as this is exploitable in terms of sending requests to auto-unblock your account thus, spending our mails per day on our mailing provider and effectively leaving us without mails for the rest of the customers which would need that.